python/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "PyGObject-Ygg-stubs"
+authors = [
+    {name = "CSI Client Tools", email = "csi-client-tools-bugs@redhat.com" },
+]
+description = "Stub files for Ygg GObject"
+requires-python = ">=3.7"
```

Comment:
would it be possible to make this work also with py3.6, which is what we have in rhel8?